pallets/funding/src/mock.rs

```diff
@@ -20,45 +20,46 @@
 
 use super::*;
 use crate as pallet_funding;
+use crate::traits::ProvideAssetPrice;
 use frame_support::{
 	construct_runtime,
 	pallet_prelude::Weight,
 	parameter_types,
-	traits::{AsEnsureOriginWithArg, ConstU16, ConstU32, ConstU64, WithdrawReasons},
+	traits::{AsEnsureOriginWithArg, ConstU16, ConstU32, ConstU64, Everything, OriginTrait, WithdrawReasons},
 	PalletId,
 };
 use frame_system as system;
-use frame_system::EnsureRoot;
-use polimec_common::credentials::EnsureInvestor;
+use frame_system::{EnsureRoot, RawOrigin as SystemRawOrigin};
+use polimec_common::{credentials::EnsureInvestor, USD_DECIMALS};
+use polkadot_parachain_primitives::primitives::Sibling;
 use sp_arithmetic::Percent;
 use sp_core::H256;
 use sp_runtime::{
-	traits::{BlakeTwo256, ConvertInto, IdentityLookup},
+	traits::{BlakeTwo256, ConvertBack, ConvertInto, Get, IdentityLookup, TryConvert},
 	BuildStorage,
 };
 use sp_std::collections::btree_map::BTreeMap;
 use std::cell::RefCell;
 use system::EnsureSigned;
+use xcm_builder::{EnsureXcmOrigin, FixedWeightBounds, ParentIsPreset, SiblingParachainConvertsVia};
+use xcm_executor::traits::XcmAssetTransfers;
 
-type Block = frame_system::mocking::MockBlock<TestRuntime>;
+pub const PLMC: Balance = 10u128.pow(PLMC_DECIMALS as u32);
+pub const MILLI_PLMC: Balance = PLMC / 10u128.pow(3);
+pub const MICRO_PLMC: Balance = PLMC / 10u128.pow(6);
+pub const EXISTENTIAL_DEPOSIT: Balance = 10 * MILLI_PLMC;
+pub const USD_UNIT: Balance = 10u128.pow(USD_DECIMALS as u32);
 
+pub type Block = frame_system::mocking::MockBlock<TestRuntime>;
```

Comment:
Where do we need this Block for? Is it just a move?

Reply:
Yes it moved and I added the pub keyword